app/src/main/res/values/preference_keys.xml

```diff
@@ -36,4 +36,5 @@
     <!-- Search Settings -->
     <string name="pref_key_show_search_suggestions" translatable="false">pref_key_show_search_suggestions</string>
 
+    <string name = "pref_key_bounce_quick_action" translatable="false">pref_key_bounce_quick_action</string>
```

Comment:
Add translation comment.

Reply:
This is a preference key and won't be translated, so I don't think that's necessary, unless I'm missing something here?

Reply:
Oh I missed that. Sorry.